src/sentry/search/utils.py

```diff
@@ -468,6 +468,40 @@ def _run_latest_release_query(
     return [row[0] for row in cursor.fetchall()]
 
 
+def get_terminal_release_for_group(
```

Comment:
Might be nice to have a short explainer for what this function does since the "terminal" naming doesn't make it immediately obvious (at least to me)

Reply:
Yeah I'm thinking maybe I should just call this `get_first_last_release_for_group`.